ci/prow/make_config.go

```diff
@@ -322,28 +328,117 @@ func combineSlices(a1 []string, a2 []string) []string {
 	return res
 }
 
-// Consolidate whitelisted and skipped branches with newly added whitelisted/skipped
+// intersectSlices returns intersect of 2 slices
+func intersectSlices(a1 []string, a2 []string) []string {
+	var res []string
+	s1 := sets.NewString(a1...)
+	for _, e2 := range a2 {
+		if s1.Has(e2) {
+			res = append(res, e2)
```

Comment:
The result can have duplicate elements, is it fine?
E.g., if a1 has one "s1", and a2 has two "s1", then the result will contain two "s1".